lib/tasks/templates.js

```diff
@@ -8,10 +8,14 @@ module.exports = function(gulp, config) {
 
 		var jade = require('jade')
 		var gulpJade = require('gulp-jade')
+		var handlebars = require('handlebars')
+		var gulpHandlebars = require('gulp-static-handlebars')
```

Comment:
Which reasons led you to use this library instead of https://github.com/shannonmoeller/gulp-hb (for ex.)?